packages/strapi-utils/lib/relations.js

```diff
@@ -1,8 +1,16 @@
 'use strict';
 
+const { prop } = require('lodash/fp');
+
 const MANY_RELATIONS = ['oneToMany', 'manyToMany', 'manyWay'];
+const RF_RELATIONS = ['oneToMany', 'manyToMany', 'manyWay', 'manyToOne', 'oneWay', 'oneToOne'];
+
+const getRelationalFields = modelDef => {
+  return modelDef.associations.filter(a => RF_RELATIONS.includes(a.nature)).map(prop('alias'));
```

Comment:
might need to handle the case a model doesn't have associations ? not sure it is initialized correctly in the mount models yet